Sources/SWBTestSupport/TestWorkspaces.swift

```diff
@@ -1028,6 +1031,8 @@ package final class TestStandardTarget: TestInternalTarget, Sendable {
                 return "lib\(name).a"
             case .objectFile:
                 return "\(name).o"
+            case .objectLibrary:
+                return "\(name).objlib"
```

Comment:
Is .objlib what CMake uses? Doesn't particularly matter, just wondering if there's any prior art at all

Reply:
No precedent, I just made this up